pkg/clientaccess/clientaccess.go

```diff
@@ -17,7 +17,8 @@ import (
 )
 
 var (
-	insecureClient = &http.Client{
+	ErrServiceUnavailable = &http.ProtocolError{ErrorString: "service unavailable"}
```

Comment:
Have to point out that this error type is deprecated and its causing a warning in my IDE 😓

Not a blocker, but worth cleaning up at some poitn

Reply:
Oh that's irritating. I was trying to be somewhat standards-conformant by using the http package's error type.

https://go-review.googlesource.com/c/go/+/33096/3/src/net/http/request.go

They deprecated it, but don't appear to actually have any plans to replace it or even stop using it. I can switch it to a more generic error since it doesn't really matter what it is.

Reply:
This has been removed.